conda-forge.yml

```diff
@@ -1,12 +1,7 @@
-appveyor:
-  secure:
-    BINSTAR_TOKEN: tumuXLL8PU75WMnRDemRy02ruEq2RpNxeK3dz0MjFssnosPm2v4EFjfNB4PTotA1
 compiler_stack: comp7
 max_py_ver: '37'
 max_r_ver: '35'
-travis:
-  secure:
-    BINSTAR_TOKEN: EjLNlquQ2jFVsxUtTcp3E50C0xUcTGYAW6al0PR1Qpxz5vs6jSpWLizx03XloiVwD4K5ykTAwe2FBNJ3Oh8uPIsXRldFzKQsr7RT+mzO9WjTakg/RD45EedEgQPYTtmBfT4etPnpOKX5l1GGSoxXgem3MkWzlPU2nbuXrXo9uHGnw6XdNBu5QNHEn81g8Ih9fNK85K7VD4s2vAdzP3EDvxbOl16H12UskCIvIsmfh/J5jjA1K1R1YFiXf+Lb99y5oCd9DDjUAUNx/dmGmS2cJBdvP45etS7W3Ddk7tIvHR7SvIvG052w7GAbkAsgUTbHST0pLsiaDZxRZlziqjUt+grFXejYhUhdBJfoJq4MXyiWF93WxEO9+BoDJQYmefqHibiS8g3tjdELhPcTdUfCUZeB9PXy1qlCCu440oid373KfIpgAsPm7OMGhGUASfkX9C1mZIgjBaU/Il32TnNxlzD18CTPbMgehElirvet7JCSV07CJuEJiZRqSE+t1aXiMMVukVDH+akW5RWF55yGzJAX8pICivglhoG7XdjGvKj0a+HjOXeoFLIRV07BiP+acZUdar8FbmPCnlX0t5nilTqp9cQAE8xXS09mTz6dQ9LEc8T8Bm1xGVOJ8SfxVwhsogIFGmNs0Cq7i2CF9VbwmNiIq+P//skvYfnuEA/PUJE=
```

Comment:
Please keep this. You need this for ppc64le builds.

Reply:
OK thanks